src/source-af-xdp.c

```diff
@@ -824,15 +829,62 @@ static TmEcode ReceiveAFXDPLoop(ThreadVars *tv, void *data, void *slot)
     SCReturnInt(TM_ECODE_OK);
 }
 
+/**
+ * \brief function to unload an AF_XDP program
+ *
+ */
+static void RunModeAFXDPRemoveProg(void)
+{
+    const char *live_dev = NULL;
```

Comment:
this can be consolidated into a single line `const char *live_dev = LiveGetDeviceName(0);`